react/features/base/known-domains/middleware.js

```diff
@@ -1,7 +1,8 @@
 // @flow
 
-import { APP_WILL_MOUNT, getDefaultURL } from '../../app';
+import { getDefaultURL } from '../../app';
```

Comment:
This makes me slightly uneasy, since we are importing this from a base feature and app is not a base feature. After the split, the function remains in features/app because base has no knowledge of a default URL of any kind. Food for thought.